pages/desktop/frontend/search.py

```diff
@@ -111,6 +113,14 @@ def click_search_result(self, count):
 
             return Detail(self.selenium, self.page.base_url).wait_for_page_to_load()
 
+        def validate_filter_sort_by_rating(self):
+            for result in self.extensions:
+                assert getattr(result, 'rating') > 4
+
+        def validate_filter_sort_by_users(self):
+            results = [getattr(result, 'users') for result in self.extensions]
+            assert sorted(results, reverse=True) == results
```

Comment:
You should return this assert statement

Reply:
Ok

Reply:
Same as above, I can move this two lines into the test and it would be less complicated.